docs/Reference/plugins/community-plugins/canonical_es.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Complemento Canónico
+published: true
+lang: es
+position: 100
+---
+
+# Complemento `canonical`
+
+<div class="docs-link_table">
+  <a class="homepage" href="https://github.com/pjlamb12/scully-plugin-canonical"></a>
+  <a class="repository" href="https://github.com/pjlamb12/scully-plugin-canonical"></a>
+</div>
+
+## Visión general
+
+El proposito del complemento canónico de de Scully es tomar un atributo link canónico de la ruta y replazar el valor de ese enlace canónico en el encabezado de la página, o agregar la etiqueta link si no existe. Esto es necesario para propósits de SEO, si un artículo se publicó primero en otro sitio, por ejemplo.
+
+## Caraterísticas
+
+- ✅ Reemplaza el link canónico con una url con la url provista para el tag link
```

Comment:
Could you try to add sense to this line?. Is hard to understand.

Reply:
done